src/db/migrations.ts

```diff
@@ -137,7 +137,7 @@ const migrations: Record<string, Migration> = {
       await db
         .updateTable("proposals")
         .where("type", "is", null)
-        .set("type", ProposalType.POST_LABEL)
+        .set("type", ProposalType.LABEL)
         .execute();
     },
```

Comment:
If we just change the `ProposalType` in the code, we could end up misreading the existing rows in the DB, & i'm not sure what issues that might cause downstream.  adding this migration will go through and update any existing rows to mitigate that potential issue

```suggestion
await db
.updateTable("proposals")
.where("type", "is", null)
        .set("type", ProposalType.LABEL)
.execute();
},
"008": {
  async up(db: Kysely<DatabaseSchema>) {
    await db
        .updateTable("proposals")
        .where("type", "is", "post_label")
        .set("type", ProposalType.LABEL)
        .execute();
  }
},
```